### PR TITLE
Desktop/MI-10-MEAN/mayank assignment-6

### PR DESCRIPTION
**Pull request by mayank30041995**

### CODE REVIEW

File: mayank/Assignment-6.1/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+  </head>
+
```
Comment (VIP5614) [inline mayank/Assignment-6.1/index.html:7]: remove extra line

File: mayank/Assignment-6.1/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <body>
+    <h2 id="myHeader">MI-9 intern Html assignment</h2>
+    <hr />
+    <p id="myHeader" style="padding: 5px"><b>Personal Details</b></p>
+    <form>
+      <label for="lname"><strong>First Name: </strong></label>
+      <input
```
Comment (VIP5614) [inline mayank/Assignment-6.1/index.html]: input tag ko aap ak line main end karo na like <input type="text" id="fname" placeholder="First Name"

File: mayank/Assignment-6.1/index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<html>
+  <head>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+  </head>
+
+  <body>
+    <h2 id="myHeader">MI-9 intern Html assignment</h2>
+    <hr />
+    <p id="myHeader" style="padding: 5px"><b>Personal Details</b></p>
+    <form>
```
Comment (VIP5614) [inline mayank/Assignment-6.1/index.html:12]: sab ke liye aap alag alag from tag kyo bana rahe hai jo ki ak from tag se kaam ho jaye ga

File: mayank/Assignment-6.1/styles.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#myHeader {
+  background-color: rgb(180, 180, 180);
+  color: black;
+  padding: 10px;
+  text-align: right;
+}
```
Comment (VIP5614) [inline mayank/Assignment-6.1/styles.css:7]: abhi aap ne proper css bhi nahi diya hai border-radius

**Review by sanketmod — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

File: mayank/Assignment-6.1/styles.css [hunk 5]
```diff
@@ -3,4 +3,5 @@
   color: black;
   padding: 10px;
   text-align: right;
+  border-radius: 10px;
```
Comment (VIP5614) [inline mayank/Assignment-6.1/styles.css:6]: border-radius only  right and left side not all side

**Review by VIP5614 — APPROVED** [review]

**Review by VIP5614 — APPROVED** [review]

**Pull request merged**